tests/mir-opt/nrvo_miscompile_111005.rs

```diff
@@ -0,0 +1,19 @@
+// This is a miscompilation, #111005 to track
+
+// unit-test: RenameReturnPlace
+
+#![feature(custom_mir, core_intrinsics)]
+extern crate core;
+use core::intrinsics::mir::*;
+
+#[custom_mir(dialect = "runtime", phase = "initial")]
+pub fn wrong(arg: char) -> char {
+    mir!({
+        let temp = arg;
+        RET = temp;
+        temp = 'b';
+        Return()
+    })
+}
+
+fn main() {}
```

Comment:
Could you add the EMIT_MIR directive?

Reply:
/me wonders how I managed to forget that